docs/source/guides/session_limits.rst

```diff
@@ -0,0 +1,75 @@
+How To Set Orchestrator Session Limits
+======================================
+
+What Are Orchestrator Sessions?
+******************************
+
+By default an Orchestrator node can handle a maximum of **10** incoming livestreams at a time. Whenever the number of concurrent transcoding sessions go above this, the Orchestrator returns an ``OrchestratorCapped`` error to the Broadcaster node.
+
+Of course, the transcoding capacity and the bandwidth an Orchestrator has varies for everyone - so you can set the ``-maxSessions`` cli parameter with the correct number to fully utilize their hardware and get maximum work from the network.
+
+For example if an Orchestrator has enough resources (i.e. GPUs/CPUs/Bandwidth) to handle ``30`` streams instead you can pass it while starting up the node -
+
+::
+
+    $ livepeer -orchestrator -transcoder ... -maxSessions 30
+
+How To Calcuate My Capacity?
```

Comment:
What about using the term "Session Limit" instead of capacity here since the guide name uses that term and since the CLI flags use that language as well?